invenio_oauthclient/contrib/cern_openid.py

```diff
@@ -90,20 +94,24 @@
 OAUTHCLIENT_CERN_OPENID_ALLOWED_ROLES = ["cern_user"]
 """CERN OAuth application role values that are allowed to be used."""
 
+_ACCOUNTS_REST_AUTH_VIEWS.update(
+    logout="invenio_oauthclient.contrib.cern_openid:CERNOpenIDLogoutView")
+ACCOUNTS_REST_AUTH_VIEWS = _ACCOUNTS_REST_AUTH_VIEWS
```

Comment:
Can we just skip the whole `_ACCOUNTS_REST_AUTH_VIEWS`? I think it is just an alias, so we could just 

```
from invenio_accounts.config import ACCOUNTS_REST_AUTH_VIEWS
    
ACCOUNTS_REST_AUTH_VIEWS.update(
    logout="invenio_oauthclient.contrib.cern_openid:CERNOpenIDLogoutView")
```

Reply:
I had problems with updating the config variables in this way in the past, they were not overwritten correctly

Reply:
There might be a problem here: it looks like that `ACCOUNTS_REST_AUTH_VIEWS` is global. So if you have configured multiple login method, e.g. local accounts, GitHub and CERN, they will all use this class.
@zzacharo can you maybe give your advice?
No idea how to solve it at the moment...

Reply:
Why don't you decouple this behaviour in the UI. For example:

- SPA logout from accounts
- SPA logout from OAuth